clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -2408,11 +2436,11 @@ void handleFailure(Throwable throwable) {
 
             @Override
             void handleFailure(Throwable throwable) {
-                future.completeExceptionally(throwable);
+                listFuture.completeExceptionally(new ApiException("Cannot find the brokers to query consumer listings."));
```

Comment:
Should we chain the caught exception?

Reply:
Was this comment addressed? I just want to make sure we're not losing the initial exception.

Reply:
Ack, sorry.